src/pybind/mgr/dashboard/model/feedback.py

```diff
@@ -0,0 +1,47 @@
+# # -*- coding: utf-8 -*-
+from enum import Enum
+
+
+class Feedback:
+    project_id: int
+    tracker_id: int
+    subject: str
+    description: str
+    status: int
+
+    class Project(Enum):
+        dashboard = 46
+        block = 9  # rbd
+        object = 10  # rgw
+        file_system = 13  # cephfs
+        ceph_manager = 46
+        orchestrator = 42
+        ceph_volume = 39
+        core_ceph = 36  # rados
+
+    class Tracker(Enum):
```

Comment:
Don't forget to adapt this according to the comments https://github.com/ceph/ceph/pull/42090#discussion_r693726026.
```suggestion
    class TrackerType(Enum):
```